src/components/ArticleCard.jsx

```diff
@@ -1,11 +1,16 @@
-const ArticleCard =({article})=>{
+import React from "react";
+import { Link } from 'react-router-dom';
 
-    return(
+const ArticleCard = ({ article }) => {
+    return (
         <li className="article-card">
```

Comment:
This cannot be a list item, you have an error in the terminal of li nested in an li. Because this component is being rendered in article list inside a list item. So either change the li tag here for something else, or in article list just return this component in your map remembering to give it a key.